app/package.json

```diff
@@ -66,6 +67,7 @@
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
     "@types/react-router-dom": "^5.1.3",
-    "@types/uuid": "^7.0.2"
+    "@types/uuid": "^7.0.2",
+    "prettier": "2.0.5"
```

Comment:
64adc7328fdd5c57a17b7375e1f98af03fb8b01a fixes app formatting introduced by this explicit dev dependency; I think it was being pulled in via another dependency before and the version changed.